### DISCUSSION TITLE
Curation report Feb 19-26

### DISCUSSION THREAD

**Discussion by andrawaag** [wikipathways/GPML-Repo#117] [Curation reports]
I reviewed the curation pipeline and edited the different pathways according to the suggestions. 
Most efforts were about (re)setting the approved status. 

I have deleted a few pathways that had the "deletion requested"-tag set. 
In two cases the documentation actually described that these pathways should not be deleted, however since the tags were set, I assumed that the tags had priority.

Furthermore, there was not much worth mentioning here.

**Comment by khanspers** [comment]
Thanks @andrawaag. Yes, the pathways you deleted were newly created tutorial pathways, and since the starter pathway has that comment in the Description about not deleting it (referring to the starter pathway), any resulting tutorial pathways have the same comment. I updated the Academy instructions to remove the "do not delete...." comment from the description to avoid confusion.
